syntax = "proto3";

package user;

service UserService {
    rpc CreateUser (UserRequest) returns (UserResponse);
}

message UserRequest {
    string email = 1;
    string password = 2;
}

message UserResponse {
    string message = 1;
    bool success = 2;
}
